apps/api/v2/src/modules/auth/guards/organization/organization-id.guard.ts

```diff
@@ -0,0 +1,20 @@
+import { OrganizationsRepository } from "@/modules/organizations/organizations.repository";
+import { Injectable, CanActivate, ExecutionContext, ForbiddenException } from "@nestjs/common";
+
+@Injectable()
+export class OrganizationIdGuard implements CanActivate {
```

Comment:
Ensures that the organization id that's being passed in the params actually belong to an organization and not a normal team.